plugins/modules/ipa_pwpolicy.py

```diff
@@ -64,6 +64,26 @@
     lockouttime:
         description: Period (in seconds) for which users are locked out.
         type: str
+    gracelimit:
+        description: Maximum number of ldap logins after password expiration.
+        type: int
+        version_added: 8.2.0
+    maxrepeat:
+        description: Maximum number of same consecutive characters.
+        type: int
+        version_added: 8.2.0
+    maxsequence:
+        description: The max. length of monotonic character sequences (abcd).
```

Comment:
```suggestion
        description: The maximal length of monotonic character sequences (for example C(abcd) is a monotonic character sequence of length 4).
```

Reply:
I reword these descriptions according Freeipa document, [this link](https://freeipa.readthedocs.io/en/latest/designs/libpwquality.html#libpwquality-features-to-be-exposed)